include/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.hpp

```diff
@@ -199,10 +204,18 @@ class Segment
     /// @return Dynamics
     Array<Shared<Dynamics>> getDynamics() const;
 
+    /// @brief Get coast dynamics array
+    /// @return Coast dynamics array
+    Array<Shared<Dynamics>> getCoastDynamics() const;
```

Comment:
I think it's misleading to call these `coastDynamics`. They're dynamics that are applied during a maneuver as well.
If you want to segment it in this way, I would maybe call it `generalDynamics` or `commonDynamics` or `freeDynamics` and `thrusterDynamics` or something like that.